### ISSUE TITLE
Compartment location bug on load

### ISSUE THREAD

**Issue by ugurdogrusoz** [iVis-at-Bilkent/newt#236]
When we load this document (and allow compound resize enabled):
[iNOS_SBGN-ED.sbgn.txt](https://github.com/iVis-at-Bilkent/newt/files/1783776/iNOS_SBGN-ED.sbgn.txt)

We expect to get this:
![image](https://user-images.githubusercontent.com/3874988/37017305-3d5e2d92-2121-11e8-8413-88f673c447ca.png)

But we get this instead:
![inos_sbgn-ed](https://user-images.githubusercontent.com/3874988/37017312-469886f0-2121-11e8-95c5-915050f280f3.png)

Plasma Membrane is not where we expect it (shifted to the right).

**Comment by hasanbalci** [comment]
@ugurdogrusoz I couldn't reproduce this bug, can you verify whether it is still valid?

**Comment by ugurdogrusoz** [comment]
Try enabling both "Allow Compound Resize" and "Infer Nesting on Load" before loading.

**Comment by ugurdogrusoz** [comment]
@hasanbalci What's your take on this issue?

**Comment by hasanbalci** [comment]
Normally, processes do not have compartment reference in this file. When we set "Infer Nesting on Load", each process is assigned to a compartment according to its position. The two processes on the right part of the Plasma Membrane are also assigned to Plasma Membrane with this option. Since "Allow Compound Resize" is also set, compartment rearranges its position by considering these two processes. This is observed only on Plasma Membrane because the processes in other compartments stay inside the other nodes. 
In short, this is an expected behavior with these options.
